nautobot/docs/user-guide/feature-guides/contact-and-team.md

```diff
@@ -0,0 +1,52 @@
+# Contacts and Teams
+
+## How to Assign Contacts and Teams to Nautobot Objects
+
+In order to assign contacts/teams to a particular object, we first need to navigate to the detail view of the object. We will use a circuit in this example.
```

Comment:
Would be good perhaps to point out that you need to define relevant Status and Role objects for the `contact association` content-type first?